test/Integration/SqlTriggerBindingIntegrationTests.cs

```diff
@@ -471,6 +475,27 @@ public void ChangeTrackingNotEnabledTriggerTest()
                 "Could not find change tracking enabled for table: 'dbo.Products'.");
         }
 
+        /// <summary>
+        /// Tests that the GetMetrics call works correctly.
+        /// </summary>
+        /// <remarks>We call this directly since there isn't a way to test scaling locally - with this we at least verify the methods called don't throw unexpectedly.</remarks>
+        [Fact]
+        public async void GetMetricsTest()
+        {
+            this.SetChangeTrackingForTable("Products");
+            IConfiguration configuration = new ConfigurationBuilder().Build();
+            var listener = new SqlTriggerListener<Product>(this.DbConnectionString, "dbo.Products", "func-id", new Moq.Mock<ITriggeredFunctionExecutor>().Object, new Moq.Mock<ILogger>().Object, configuration);
```

Comment:
Can add namespace `using Moq`. Also `Mock<X>.Object` can be simplified to `Mock.Of<X>()`.